about/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en" id="about">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial scale=1">
+    <link rel="stylesheet" href="/styles/style.css">
+    <link rel="stylesheet" href="/styles/responsive.css">
+    <title>Welcome to my Paradox Collection!</title>
+  </head>
+
+  <body>
+    <header>
+      <div>
+        <a href="/">
+          <img class="logo"
+            src="/assets/nakin.jpeg"
+            width="100" height="100"
+            alt="Drawing of an owl"
+          >
+        </a>
+      </div>
+
+      <menu class="index">
+        <img class="hamburger-menu"
+          src="/assets/hamburger-menu.svg"
+          width="40" height="40"
+          alt="Hamburger menu"
+        >
+        <div class="index-title">
+          <span>Uncanny Index ↓</span>
+        </div>
+        <ul class="dropdown-content">
+          <li><a href="/subpages/self-fulfilling-prophesy">Self Fulfilling Prophecy</a></li>
```

Comment:
You need to remove `/subpages` from everything, turning 

```
/subpages/self-fulfilling-prophesy
```

Into

```
/self-fulfilling-prophesy
```

Sounds like a lot but it is a good opportunity to learn your editor's (or a linux console tool's) search and replace feature :P